### PR TITLE
Add a new UI library for Tailwind CSS: fit the contribution guidelines.

### PR DESCRIPTION
**Pull request by sidiDev**
| Name                 | Link                 |
| -------------------- | -------------------- |
Float UI                floatui.com

I just added a new UI ibrary for Tailwind CSS in UI Libraries, Components & Templates category, and hope you like it.

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by innocenzi — CHANGES_REQUESTED** [review]
Please use an actual URL as the link, shorten the description if possible, fix the brand names in your description, place the project at the end of the emoji category and update the PR title to fit the contribution guidelines.

**Comment by sidiDev** [comment]
> Please use an actual URL as the link, shorten the description if possible, fix the brand names in your description, place the project at the end of the emoji category and update the PR title to fit the contribution guidelines.

**Comment by sidiDev** [comment]
> Please use an actual URL as the link, shorten the description if possible, fix the brand names in your description, place the project at the end of the emoji category and update the PR title to fit the contribution guidelines.

Hi @innocenzi ! That's done, you can check it out now.

**Comment by joshmanders** [comment]
@sidiDev 
> Hi @innocenzi ! That's done, you can check it out now.

You closed the PR.

**Comment by sidiDev** [comment]
> @sidiDev
> 
> > Hi @innocenzi ! That's done, you can check it out now.
> 
> You closed the PR.

Yeah, you're right, I'm sorry for that, it was a mistake, I reopen it.

**Comment by joshmanders** [comment]
I just noticed this was still open. If it looks good to you @innocenzi I'll merge.

File: README.md [hunk 1]
```diff
@@ -304,6 +304,7 @@
 - 📁 [Material Tailwind Dashboard React](https://www.creative-tim.com/product/material-tailwind-dashboard-react) - Free Tailwind CSS and React admin template.
 - 📁 [Admin One Vue 3](https://github.com/justboil/admin-one-vue-tailwind) - Free Vue.js 3 Tailwind CSS admin template with Vite & Vue CLI support.
 - 📁 [Cruip](https://cruip.com/) - Beautifully designed HTML, React, and Vue.js templates.
+- 🧩 [Float UI](https://floatui.com) - UI components with Tailwind even with pure CSS, support both directions LTR and RTL.
```
Comment (innocenzi) [inline README.md:307]:
```suggestion
- 🧩 [Float UI](https://floatui.com) - UI components with Tailwind CSS even with pure CSS, with LTR and RTL support.
```

**Review by innocenzi — APPROVED** [review]

**Review by innocenzi — CHANGES_REQUESTED** [review]
I updated the description - but please move the entry below other components (Daisy UI)

**Comment by joshmanders** [comment]
@sidiDev address the request by @innocenzi and we'll get this merged in.

**Comment by joshmanders** [comment]
Closed because this PR was abandoned.

**Pull request closed without merging**